maxim/logger/mistral/client.py

```
@@ -0,0 +1,215 @@
+                name=generation_name,
+                model_parameters=MistralUtils.get_model_params(**kwargs),
+                messages=MistralUtils.parse_message_param(messages),
+            )
+            generation = trace.generation(gen_config)
+        except Exception as e:
+            scribe().warning(
+                f"[MaximSDK][MaximMistralChat] Error in generating content: {e}"
+            )
+
+        stream = self._chat.stream(*args, **kwargs)
+        chunks: List[dict] = []
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)